test/common/stats/stat_test_utility.h

```diff
@@ -57,54 +56,6 @@ class TestSymbolTable {
 void forEachSampleStat(int num_clusters, bool include_other_stats,
                        std::function<void(absl::string_view)> fn);
 
-// Tracks memory consumption over a span of time. Test classes instantiate a
-// MemoryTest object to start measuring heap memory, and call consumedBytes() to
-// determine how many bytes have been consumed since the class was instantiated.
-//
-// That value should then be passed to EXPECT_MEMORY_EQ and EXPECT_MEMORY_LE,
-// defined below, as the interpretation of this value can differ based on
-// platform and compilation mode.
-class MemoryTest {
```

Comment:
can you do me a favor and leave  a `using` nickname for MemoryTest here, pointing to its new location?

We have a bunch of references to the current name in non-OSS code that we can fix to point to your new one, and then after htat rmeove the nickname. You can leave an issue assigned to me to clean this up.

Reply:
Done.